Resources/config/listeners.xml

```diff
@@ -24,6 +25,11 @@
                 <argument type="service" id="annotation_reader" />
             </call>
         </service>
+        <service id="stof_doctrine_extensions.listener.ip_traceable" class="%stof_doctrine_extensions.listener.ip_traceable.class%" public="false">
```

Comment:
please do not add a parameter for the service class. This is discouraged in Symfony now